analyzers/src/SonarAnalyzer.CSharp/Rules/DisposableNotDisposed.cs

```diff
@@ -177,11 +172,8 @@ private static void TrackAssignmentsToLocalsAndPrivateFields(SyntaxNode typeDecl
             }
         }
 
-        private static bool IsLocalOrPrivateField(ISymbol symbol)
-        {
-            return symbol.Kind == SymbolKind.Local ||
-                (symbol.Kind == SymbolKind.Field && symbol.DeclaredAccessibility == Accessibility.Private);
-        }
+        private static bool IsLocalOrPrivateField(ISymbol symbol) =>
+            symbol.Kind == SymbolKind.Local || (symbol.Kind == SymbolKind.Field && symbol.DeclaredAccessibility == Accessibility.Private);
```

Comment:
Let's format this one on two lines, because it's a combination of `||` and `&&`.
```suggestion
            symbol.Kind == SymbolKind.Local 
            || (symbol.Kind == SymbolKind.Field && symbol.DeclaredAccessibility == Accessibility.Private);
```